engines/python/setup/djl_python/huggingface.py

```diff
@@ -22,6 +22,7 @@
 from djl_python.inputs import Input
 from djl_python.outputs import Output
 from djl_python.streaming_utils import StreamingUtils
+from djl_python.rolling_batch import RollingBatch
```

Comment:
Could we not using HuggingFace handler, since you already introducing many other options here. Just create a new rolling entrypoint. @frankfliu thoughts?

Reply:
My suggestion is all traditional pieces stay here, where rolling batch comes to a new chapter